reconcile/openshift_resources_base.py

```diff
@@ -577,12 +577,10 @@ def fetch_provider_vault_secret(
 
     # populate data
     for k, v in raw_data.items():
-        if v == "":
-            continue
         if k.lower().endswith(QONTRACT_BASE64_SUFFIX):
             k = k[: -len(QONTRACT_BASE64_SUFFIX)]
             v = v.replace("\n", "")
-        elif v is not None:
+        elif v != "":
```

Comment:
`base64_encode_secret_field_value` function can be reused here so no duplicate logic.

same [here](https://github.com/app-sre/qontract-reconcile/pull/3829/files#diff-e1ae1861e6058b02d61980281bb71def45745bdfc82c269bfcf2a52f6be4fabdL424)